src/makeCalculator.js

```diff
@@ -37,7 +37,34 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function(initValue, argValue) {
```

Comment:
why it in such form here and in other cases not?

Reply:
you're right, it could be shortened